<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ferramenta para quebrar a maldição</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btClick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Recitar feitiço!</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Há muito tempo um poderoso mago da computação amaldiçoou a todos que desejam entrar nesse fantástico mundo do desenvolvimento, e a única forma de quebrar esse feitiço é escrevendo um programa que recite as palavras mágicas necessárias para quebrar a maldição e assim poder aprender a linguagem que está sendo estudada.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbHelloWorld">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbHide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Muito bem, agora com o feitiço quebrado podemos dar continuidade a nossos estudos!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
